Bottom: 7b53da29242a2e954635f03e8d5edf636cfec579
Top:    ffbd44d04a08775494b692404a5f60f79bcf2fe6
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-07-20 11:38:38 +0800

Refresh of tuntap-batched-sendmsg

---

diff --git a/drivers/net/tun.c b/drivers/net/tun.c
index 6691b9d..4dc61fd 100644
--- a/drivers/net/tun.c
+++ b/drivers/net/tun.c
@@ -2493,7 +2493,7 @@ static int tun_sendmsg(struct socket *sock, struct msghdr *m, size_t total_len)
 		rcu_read_lock();
 
 		for (i = 0; i < n; i++) {
-			xdp = &((struct xdp_buff *)ctl->ptr)[i]);
+			xdp = &((struct xdp_buff *)ctl->ptr)[i];
 			tun_xdp_one(tun, tfile, xdp);
 		}
